### ISSUE TITLE
ValueError: too many values to unpack

### ISSUE THREAD

**Issue by steveoh** [pwaller/pyprof2calltree#14]
I'm using windows and python 2.7.5

I receive `ValueError: too many values to unpack` when trying to run the code you have in your readme. 

```python
visualize(profiler.getstats())                            # run kcachegrind
convert(profiler.getstats(), 'profiling_results.kgrind')  # save for later
```

stack trace
```
File "C:\Python27\ArcGISx6410.2\lib\site-packages\pyprof2calltree.py", line 324, in convert
    converter.output(f)
  File "C:\Python27\ArcGISx6410.2\lib\site-packages\pyprof2calltree.py", line 133, in output
    self._entry(entry)
  File "C:\Python27\ArcGISx6410.2\lib\site-packages\pyprof2calltree.py", line 208, in _entry
    for subentry, call_info in calls:
ValueError: too many values to unpack
```

printing `profile.getstats()` returns an array like `[_lsprof.profiler_entry(code=<code object __init__ at 0000000001E0EE30, file ...`

**Comment by deleteman** [comment]
I can confirm that the simple example from the website doesn't work with python 2.7.6 and the latest version of pyprof2calltree (1.3.2).
Is this project still being maintained?

Sample code:
```
from xml.etree import ElementTree
from cProfile import Profile
xml_content = '<a>\n' + '\t<b/><c><d>text</d></c>\n' * 100 + '</a>'
profiler = Profile()
profiler.runctx(
"ElementTree.fromstring(xml_content)",
locals(), globals())

from pyprof2calltree import convert, visualize
convert(profiler.getstats(), 'profiling_results.kgrind')  # fails but saves file
visualize(profiler.getstats())                            # fails
```
Error:

```
Traceback (most recent call last):
  File "pyprof2calltree-test.py", line 10, in <module>
    convert(profiler.getstats(), 'profiling_results.kgrind')  # save for later
  File "/usr/local/lib/python2.7/dist-packages/pyprof2calltree.py", line 324, in convert
    converter.output(f)
  File "/usr/local/lib/python2.7/dist-packages/pyprof2calltree.py", line 133, in output
    self._entry(entry)
  File "/usr/local/lib/python2.7/dist-packages/pyprof2calltree.py", line 208, in _entry
    for subentry, call_info in calls:
ValueError: too many values to unpack
```

**Comment by pwaller** [comment]
Yes. However, I am not the original author. Patches are welcome and I will make releases as quickly as I can. But I don't have a lot of spare time to dig into problems I can't immediately understand right now. Apologies. If anyone can help me understand the problem or contribute fixes it would be greatly appreciated!

On 28 January 2015 21:48:11 GMT+00:00, Fernando Doglio <notifications@github.com> wrote:
>I can confirm that the simple example from the website doesn't work
>with python 2.7.6 and the latest version of pyprof2calltree (1.3.2).
>Is this project still being maintained?
>
>Sample code:
>```
>from xml.etree import ElementTree
>from cProfile import Profile
>xml_content = '<a>\n' + '\t<b/><c><d>text</d></c>\n' * 100 + '</a>'
>profiler = Profile()
>profiler.runctx(
>"ElementTree.fromstring(xml_content)",
>locals(), globals())
>
>from pyprof2calltree import convert, visualize
>convert(profiler.getstats(), 'profiling_results.kgrind')  # fails but
>saves file
>visualize(profiler.getstats())                            # fails
>```
>Error:
>
>```
>Traceback (most recent call last):
>  File "pyprof2calltree-test.py", line 10, in <module>
>convert(profiler.getstats(), 'profiling_results.kgrind')  # save for
>later
>File "/usr/local/lib/python2.7/dist-packages/pyprof2calltree.py", line
>324, in convert
>    converter.output(f)
>File "/usr/local/lib/python2.7/dist-packages/pyprof2calltree.py", line
>133, in output
>    self._entry(entry)
>File "/usr/local/lib/python2.7/dist-packages/pyprof2calltree.py", line
>208, in _entry
>    for subentry, call_info in calls:
>ValueError: too many values to unpack
>```
>
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/pwaller/pyprof2calltree/issues/14#issuecomment-71923394

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by deleteman** [comment]
Well, it's not a fix for the problem, but a minor change on the example, solves the issue:
```python
from xml.etree import ElementTree
from cProfile import Profile
import pstats
xml_content = '<a>\n' + '\t<b/><c><d>text</d></c>\n' * 100 + '</a>'
profiler = Profile()
profiler.runctx(
"ElementTree.fromstring(xml_content)",
locals(), globals())

from pyprof2calltree import convert, visualize
stats = pstats.Stats(profiler) #use pstats.Stats instead of profiler.getstats()
visualize(stats)                        # run kcachegrind
convert(stats, 'profiling_results.kgrind')  # save for later
```

**Comment by pwaller** [comment]
Hm, that does not crash, but provides a rather uninteresting profile to visualise. Maybe it's not a problem, but I can't tell if it is actually working as intended.

**Comment by zbenjamin** [comment]
This should be fixed by #15

**Comment by pwaller** [comment]
Closing since it should be resolved in #15. If not, please file a new bug, thanks!
